packages/core/src/components/essentials/title.ts

```diff
@@ -6,74 +6,94 @@ import { TooltipTypes, Events } from "./../../interfaces";
 export class Title extends Component {
 	type = "title";
 
+	render() {
+		const svg = this.getContainerSVG();
+
+		const text = svg.selectAll("text.title")
+			.data([this.model.getOptions().title]);
+
+		text.enter()
+			.append("text")
+			.classed("title", true)
+			.merge(text)
+			.attr("x", 0)
+			.attr("y", "1em")
+			.html(d => d);
+
+		// check the max space the title has to render
+		const maxWidth = this.getMaxTitleWidth();
+		const title = DOMUtils.appendOrSelect(svg, "text.title");
+
+		// check if title needs truncation (and tooltip support)
+		if (title.node().getComputedTextLength() > maxWidth) {
```

Comment:
Should we also check if it's `> 0`?
```suggestion
		if (title.node().getComputedTextLength() > maxWidth > 0) {
```